addons/account/migrations/13.0.1.1/post-migration.py

```diff
@@ -0,0 +1,779 @@
+# Copyright 2020 ForgeFlow <http://www.forgeflow.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+def fill_account_reconcile_model_second_analytic_tag_rel_table(env):
+    openupgrade.logged_query(
+        env.cr, """
+        INSERT INTO account_reconcile_model_second_analytic_tag_rel (
+            account_reconcile_model_id, account_analytic_tag_id)
+        SELECT account_reconcile_model_id, account_analytic_tag_id
+        FROM account_reconcile_model_analytic_tag_rel
+        """,
+    )
+
+
+def type_change_account_reconcile_model_taxes(env):
+    openupgrade.m2o_to_x2m(
+        env.cr,
+        env['account.reconcile.model'], 'account_reconcile_model',
+        'second_tax_ids', openupgrade.get_legacy_name('second_tax_id')
+    )
+    openupgrade.m2o_to_x2m(
+        env.cr,
+        env['account.reconcile.model'], 'account_reconcile_model',
+        'tax_ids', openupgrade.get_legacy_name('tax_id')
+    )
+    openupgrade.m2o_to_x2m(
+        env.cr,
+        env['account.reconcile.model.template'],
+        'account_reconcile_model_template',
+        'second_tax_ids', openupgrade.get_legacy_name('second_tax_id')
+    )
+    openupgrade.m2o_to_x2m(
+        env.cr,
+        env['account.reconcile.model.template'],
+        'account_reconcile_model_template',
+        'tax_ids', openupgrade.get_legacy_name('tax_id')
+    )
+
+
+def map_res_company_fiscalyear_last_month(env):
+    openupgrade.map_values(
+        env.cr,
+        openupgrade.get_legacy_name('fiscalyear_last_month'),
+        'fiscalyear_last_month',
+        [(x, '%s' % x) for x in range(1, 13, 1)],
+        table='res_company',
+    )
+
+
+def fill_account_fiscal_position_company_id(env):
+    openupgrade.logged_query(
+        env.cr, """
+        UPDATE account_fiscal_position afp
+        SET company_id = ru.company_id
+        FROM res_users ru
+        WHERE ru.id = afp.create_uid AND
+        afp.company_id is NULL
+        """,
+    )
+
+
+def fill_account_account_type_internal_group(env):
+    openupgrade.logged_query(
+        env.cr, """
+        UPDATE account_account_type
+        SET internal_group = 'off_balance'
+        WHERE internal_group IS NULL
+        """,
+    )
+
+
+def map_account_journal_post_at_bank_rec(env):
+    openupgrade.map_values(
+        env.cr,
+        openupgrade.get_legacy_name('post_at_bank_rec'),
+        'post_at',
+        [(True, 'bank_rec')],
+        table='account_journal',
+    )
+
+
+def fill_account_journal_restrict_mode_hash_table(env):
+    openupgrade.logged_query(
+        env.cr, """
+        UPDATE account_journal
+        SET restrict_mode_hash_table = TRUE
+        WHERE {} != TRUE
+        """.format(openupgrade.get_legacy_name('update_posted'))
+    )
```

Comment:
Need to fill secure_sequence_id.
As I remember, I couldn`t confirm the invoice drafted<pre>
def fill_account_journal_restrict_mode_hash_table(env):
    openupgrade.logged_query(
        env.cr, """
        UPDATE account_journal
        SET restrict_mode_hash_table = TRUE
        WHERE {} != TRUE
        RETURNING id
        """.format(openupgrade.get_legacy_name('update_posted'))
    )
    ids = [x[0] for x in env.cr.fetchall()]
    if ids:
        env['account.journal'].browse(ids)._create_secure_sequence(['secure_sequence_id'])
</pre>